Settings/Model/BKDefaults.m

```diff
@@ -61,6 +61,10 @@ - (id)initWithCoder:(NSCoder *)coder
   _xCallBackURLEnabled = [coder decodeBoolForKey:@"xCallBackURLEnabled"];
   _xCallBackURLKey = [coder decodeObjectForKey:@"xCallBackURLKey"];
   _disableCustomKeyboards = [coder decodeBoolForKey:@"disableCustomKeyboards"];
+  _playSoundOnBell = [coder decodeBoolForKey:@"playSoundOnBell"];
+  _notificationOnBellUnfocused = [coder decodeBoolForKey:@"notificationOnBellUnfocused"];
+  _hapticFeedbackOnBell = [coder decodeBoolForKey:@"hapticFeedbackOnBell"];
```

Comment:
We talked about setting `_hapticFeedbackOnBell` on by default and decided to rename it to `_hapticFeedbackOnBellOff`